mysql/const.go

```diff
@@ -452,6 +452,16 @@ func (m SQLMode) HasRealAsFloatMode() bool {
 	return m&ModeRealAsFloat == ModeRealAsFloat
 }
 
+// HasNoBackslashEscapesMode detects if 'NO_BACKSLASH_ESCAPES' mode is set in SQLMode
```

Comment:
This function is not related with IGNORE_SPACE.

Reply:
@shenli  It seems to be taken into this pr in a merge. I'll delete this.